micro_ros_setup/config/zephyr/generic/client-colcon.meta

```diff
@@ -0,0 +1,36 @@
+{
+    "names": {
+        "rcl": {
+            "cmake-args": [
+                "-DBUILD_TESTING=OFF"
+            ]
+        }, 
+        "rcutils": {
+            "cmake-args": [
+                "-DENABLE_TESTING=OFF",
+                "-DRCUTILS_NO_FILESYSTEM=ON",
+                "-DRCUTILS_NO_THREAD_SUPPORT=ON",
+                "-DRCUTILS_AVOID_DYNAMIC_ALLOCATION=ON"
+            ]
+        },
+        "microxrcedds_client": {
+            "cmake-args": [
+                "-DUCLIENT_ISOLATED_INSTALL=OFF",
```

Comment:
I think it is not needed because of the _colcon.pkg_ at Micro XRCE-DDS Client.

Reply:
Done